### ISSUE TITLE
Showstopper Update store does not display info

### ISSUE THREAD

**Issue by sagsaw** [covid-maps/covid-maps#101]
On clicking update, the already stored information of the store is not displayed in the update form.

My entry 

![IMG_20200404_212751](https://user-images.githubusercontent.com/63158972/78455308-14d21500-768d-11ea-857a-a7f78d1f584b.jpg)

After clicking update ....

![Screenshot_2020-04-04-20-56-58-607_org mozilla firefox](https://user-images.githubusercontent.com/63158972/78455322-274c4e80-768d-11ea-8d0e-58474c4d4e43.jpg)

As it can be seen already save info is not displayed in update form.

**Comment by sagsaw** [comment]
Even after todays update which now has time fields, already entered information for a store is not visible in update form so it need to be entered again.

**Comment by arjun27** [comment]
@sagsaw - were you trying to edit the submission?

**Comment by sagsaw** [comment]
Yes ... Chk the screen shots ... I created a store some days back ... The. Yesterday tried to Update/Edit using the provided Button/CTA ...

EXPECTATION IN EDIT/UPDATE IS THE FORM MUST BE POPULATED WITH PREVIOUS DATA.

**Comment by sampoorna** [comment]
The update button is not the same as edit ie., it's not expected that you will be able to modify previously entered info. This is because we're not tracking what was the last update you made, and because we expect users to have new information to add, rather than to modify a single post over and over.

**Comment by arjun27** [comment]
> Yes ... Chk the screen shots ... I created a store some days back ...

thanks @sagsaw. can you tell me what info you were trying to add? I'm all for making improvements to this flow, but I want to understand what was happening:

* did you want to edit something that was wrong in the existing listing?
* or, did you want to add new info which was missing in the existing listing?

**Comment by sagsaw** [comment]
Ok ... So as you can in first screen shot ... I added timings in Useful information ...

Later when i come back again to covidmaps say 2 days later bcos now i know that i entered wrong info and i click update info ... I expect the previous info i entered to appear in the form fields .. but safety observations and useful information fields are not filled with PREVIOUS information in entered.

**Comment by sagsaw** [comment]
@sampoorna Well in that case old info will be lost which could be correct ..

Person A enters some info person B wants to  update only one field out of 5 bcos hez been to that store latest  ... in that case its n issue ..

I feel update and edit must be same ... When in update mode current info visible in card must be populated in the form fields ... And let person updating update only what he/she wants to update ... Could be location store type timings useful information or safety observations ...

**Comment by sagsaw** [comment]
@sampoorna @arjun27 Just checked the latest updated version .. the updates one after the other could be confusing ... And will become too long ... 

![Screenshot_2020-04-07-11-30-12-471_org mozilla firefox](https://user-images.githubusercontent.com/63158972/78635227-41f11400-7895-11ea-8f4a-10309989f683.jpg)

**Comment by sagsaw** [comment]
If one store which is a goto store, say like Dmart ... They keep on chamging their timings and instructions time to time .. from practical experiemce d mart in my area hss changed policies almost daily ... 

Imagine 15 updates visible in a single card.

You can have a view history popup instead in the card.
But the card must show only the latest info.
